demo-notebooks/guided-demos/2_job_client.ipynb

```diff
@@ -252,6 +258,77 @@
   {
    "cell_type": "code",
    "execution_count": null,
+<<<<<<< HEAD:demo-notebooks/guided-demos/2_job_client.ipynb
+=======
+   "id": "496139cc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "jobdef = DDPJobDefinition(\n",
+    "    name=\"mnistjob\",\n",
+    "    script=\"mnist.py\",\n",
+    "    scheduler_args={\"namespace\": \"default\"},\n",
+    "    j=\"1x1\",\n",
+    "    gpu=0,\n",
+    "    cpu=1,\n",
+    "    memMB=8000,\n",
+    "    mcad=True,\n",
+    "    image=\"quay.io/project-codeflare/mnist-job-test:v0.0.1\"\n",
+    ")\n",
+    "job = jobdef.submit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0837e43b",
+   "metadata": {},
+   "source": [
+    "Once again, we can look at job status and logs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d18d42c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.status()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "36d7ea97",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.logs()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aebf376a",
+   "metadata": {},
+   "source": [
+    "This time, once the pods complete, we can clean them up alongside any other associated resources. The following command can also be used to delete jobs early for both Ray and MCAD submission:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ebbb0674",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.cancel()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d41b90e",
+>>>>>>> 583747f (Added demo notebook for Kueue by default):demo-notebooks/guided-demos/preview_nbs/2_basic_jobs.ipynb
```

Comment:
more diff

Reply:
Good catch, that was causing to not be able to even open the file: 
![image](https://github.com/project-codeflare/codeflare-sdk/assets/73656840/40a52385-4383-4391-8d48-5c85cf639d08)